diff --git a/android/build.gradle b/android/build.gradle
index 2614c62d2188dab55b22d7a241fd07c2bc4a8a81..3593aac724d333ef3d3eef06f7115372b278756c 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -22,5 +22,5 @@ android {
 }
 
 dependencies {
-  compile "com.facebook.react:react-native:+" // From node_modules
+  implementation('com.facebook.react:react-native:+') // From node_modules
 }
